tracing-bom/build.gradle

```
@@ -42,11 +42,13 @@ micronautBom {
                         "io.zipkin.proto3"
                 ] as Set
         )
+        dependencies.add("io.opentelemetry:opentelemetry-bom:1.30.1")
+        dependencies.add("io.opentelemetry:opentelemetry-bom-alpha:1.30.1-alpha")
         dependencies.add("io.zipkin.brave:brave-bom:5.16.0")
         dependencies.add("io.zipkin.reporter2:zipkin-reporter-bom:2.16.3")
-        dependencies.add("io.opentelemetry:opentelemetry-bom:1.26.0")
-        dependencies.add("io.opentelemetry:opentelemetry-bom-alpha:1.26.0-alpha")
-        dependencies.add("io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom:1.26.0")
-        dependencies.add("io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha:1.26.0-alpha")
+        dependencies.add("io.opentelemetry:opentelemetry-bom:1.29.0")
+        dependencies.add("io.opentelemetry:opentelemetry-bom-alpha:1.29.0-alpha")
+        dependencies.add("io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom:1.29.0")
+        dependencies.add("io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha:1.29.0-alpha")
```

```suggestion
        dependencies.add("io.opentelemetry.instrumentation:opentelemetry-instrumentation-bom-alpha:1.30.0-alpha")
```

https://central.sonatype.com/artifact/io.opentelemetry.instrumentation/opentelemetry-instrumentation-bom-alpha/versions